### PR TITLE
Use collection expressions for empty collections

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
@ToddGrun ptal :)

File: src/EditorFeatures/Core/Workspaces/AbstractTextBufferVisibilityTracker.cs [hunk 1]
```diff
@@ -142,7 +142,7 @@ protected sealed class VisibleTrackerData : IDisposable
             /// The callbacks that want to be notified when our <see cref="TextViews"/> change visibility.  Stored as an
             /// <see cref="ImmutableHashSet{T}"/> so we can enumerate it safely without it changing underneath us.
             /// </summary>
-            public ImmutableHashSet<Action> Callbacks { get; private set; } = ImmutableHashSet<Action>.Empty;
```
Comment (ToddGrun) [inline src/EditorFeatures/Core/Workspaces/AbstractTextBufferVisibilityTracker.cs:145]: >ImmutableHashSet<Action>.Empty; [](http://example.com/codeflow?start=78&length=31)

Is ImmutableDictionary.Empty covered by this analyzer too? 

If so, was curious the usage in src\Features\Core\Portable\EditAndContinue\ActiveStatementsMap.cs wasn't changed.

**Review by ToddGrun — APPROVED** [review]
:shipit:

File: src/EditorFeatures/Core/Workspaces/AbstractTextBufferVisibilityTracker.cs (hunk 1 above)
Comment (CyrusNajmabadi) [reply src/EditorFeatures/Core/Workspaces/AbstractTextBufferVisibilityTracker.cs:145]: Should be. I'll check.

**Pull request merged**